app/assess/templates/flag_application.html

```diff
@@ -14,6 +14,32 @@
 
     {{ logout_partial(sso_logout_url) }}
 </div>
+
+{% if form.errors.get("justification") or form.errors.get("section") %}
+<div class="govuk-error-summary" data-module="govuk-error-summary">
+    <div role="alert">
+        <h2 class="govuk-error-summary__title">
+            There is a problem
+        </h2>
+        <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+                {% if form.errors.get("justification") %}
+                <li>
+                    <a href=#{{form.justification.id}}>Reason for flagging</a>
+                </li>
+                {% endif %}
+                {% if form.errors.get("section") %}
+                <li>
+                    <a href=#{{form.section.id}}>Section to flag</a>
+                </li>
+                {% endif %}
```

Comment:
Instead of hard-coding the errors on the front-end, can we add them to the form and then retrieve them via their keys?

I.e:

https://github.com/communitiesuk/funding-service-design-assessment/blob/e4251d9d7376bd95932e374369e94c5c4458efb0/app/assess/forms/flag_form.py#L14-L16

https://github.com/communitiesuk/funding-service-design-assessment/blob/0a2537594688f517209d9aa964b474547c049cbd/app/assess/templates/flag_application.html#L47

Reply:
Not aware of this solution. I took the example from `score.html` & implemented the same. Will do this way then.